packages/hash/frontend/src/components/layout/PageSidebar/AccountEntityTypeList/EntityTypeMenu.tsx

```diff
@@ -1,153 +1,130 @@
-                  <ListItemButton onClick={() => setOpen(false)}>
-                    <FontAwesomeIcon icon={icon} />
-                    <Typography variant="smallTextLabels">{title}</Typography>
-                  </ListItemButton>
-                </Link>
-              );
-            }
-            if (onClick) {
-              return (
-                <ListItemButton
-                  onClick={() => {
-                    onClick();
-                    setOpen(false);
-                  }}
-                  sx={{
-                    mx: 0.5,
-                    py: 1,
-                    px: 1.5,
-                    borderRadius: "4px",
-
-                    "& svg": {
-                      fontSize: 16,
-                      mr: 0.75,
-                      color: ({ palette }) => palette.gray[50],
-                    },
-
-                    "& .MuiTypography-root": {
-                      color: ({ palette }) => palette.gray[80],
-                      fontWeight: 500,
-                    },
-
-                    "&:hover": {
-                      backgroundColor: ({ palette }) => palette.gray[20],
-
-                      "& svg": {
-                        color: ({ palette }) => palette.gray[60],
-                      },
-                    },
-
-                    "&:focus": {
-                      backgroundColor: ({ palette }) => palette.blue[70],
-                    },
-                  }}
-                >
+    <Menu {...bindMenu(popupState)}>
+      {menuItems.map(({ title, icon, onClick, href, id, faded }) => {
+        if (href) {
+          return (
+            <MenuItem key={id} onClick={() => popupState.close()}>
+              <Link sx={{ display: "flex" }} key={id} noLinkStyle href={href}>
+                <ListItemIcon>
                   <FontAwesomeIcon icon={icon} />
-                  <Typography variant="smallTextLabels">{title}</Typography>
-                </ListItemButton>
-              );
-            }
-            return null;
-          })}
-        </Box>
-      </Popover>
-    </Box>
+                </ListItemIcon>
+                <ListItemText primary={title} />
+              </Link>
+            </MenuItem>
+          );
+        }
+        if (onClick) {
+          return (
+            <MenuItem
+              key={id}
+              sx={{
+                ...(Boolean(faded) && {
+                  [`& .${listItemTextClasses.primary}`]: {
+                    color: ({ palette }) => palette.gray[50],
+                  },
+                  [`& .${listItemIconClasses.root}`]: {
+                    color: ({ palette }) => palette.gray[40],
+                  },
+                }),
+              }}
+              onClick={onClick}
+            >
+              <ListItemIcon>
+                <FontAwesomeIcon icon={icon} />
+              </ListItemIcon>
+              <ListItemText primary={title} />
+            </MenuItem>
```

Comment:
Found this to be a good way for handling our menu/dropdowns. The styling (base/hover/focus/selected states) for `ListItemText` and `ListItemIcon` is in `MuiMenuItemThemeOptions`. This will be handled properly in this [task](https://app.asana.com/0/1201959586244671/1201926880408096)